Vagrantfile

```diff
@@ -0,0 +1,66 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  # config.vm.network "private_network", ip: "192.168.33.10"
+
+  config.vm.synced_folder '.', '/test'
+
+  config.vm.provision "shell", privileged: true, inline: <<-SHELL
+    apt-get update
+    apt-get install -y \
+      ruby2.0 \
+      ruby2.0-dev \
+      git \
+      make \
+      gcc \
+      lxc \
+      libcap2 \
+      libapparmor1 \
+      libselinux1 \
+      libseccomp2 \
+      libgnutls28 \
+      python3-dev \
+      lxc \
+      lxc-dev \
+
+    gem2.0 install bundler
+  SHELL
+
+  lxc_config = <<-CONFIG
+    lxc.network.type = veth
+    lxc.network.link = lxcbr0
+    lxc.network.flags = up
+    lxc.network.hwaddr = 00:16:3e:xx:xx:xx
+    lxc.id_map = u 0 100000 65536
+    lxc.id_map = g 0 100000 65536
+  CONFIG
+
+  lxc_usernet = 'vagrant veth lxcbr0 10'
+  lxc_conf = 'lxc.lxcpath = /home/vagrant/.local/share/lxc'
+
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    mkdir -p /home/vagrant/.config/lxc /home/vagrant/.local/share/lxc /home/vagrant/.cache/lxc
+    echo "#{lxc_config}" | tee /home/vagrant/.config/lxc/default.conf
+    echo "#{lxc_usernet}" | sudo tee /etc/lxc/lxc-usernet
+    echo "#{lxc_usernet}" | tee /home/vagrant/.config/lxc/lxc.conf
+  SHELL
+
+  config.vm.provision 'shell', inline: <<-SHELL
+    cd /test
+    # don't put the bundle in the mounted directory -- too slow :)
+    bundle package --all --path=/test-vendor/cache
+    bundle install --path=/test-vendor/bundle --local
+  SHELL
+
+  config.vm.provision 'shell', inline: <<-SHELL
+    if [ ! -e /home/vagrant/lxc-configured ]; then
+      touch /home/vagrant/lxc-configured
+      sudo reboot
+    fi
+  SHELL
+end
```

Comment:
I made a [gist of the vagrantfile](https://gist.github.com/margueritepd/e773692c99723ebfb3af), because I am going to use it in other projects. Feel free to link to it in the README, or not :)